python-checks/src/main/java/org/sonar/python/checks/ComparisonToNoneCheck.java

```diff
@@ -67,10 +69,6 @@ private static void addIssue(SubscriptionContext ctx, Tree tree, String comparis
     ctx.addIssue(tree, String.format("Remove this %s; it will always be %s.", comparisonKind, resultAsString));
   }
 
-  private static boolean isNone(InferredType type) {
-    return type.canOnlyBe(BuiltinTypes.NONE_TYPE);
-  }
-
   private static boolean cannotBeNone(InferredType type) {
     return !type.canBeOrExtend(BuiltinTypes.NONE_TYPE);
   }
```

Comment:
I see you have moved the method `isNone()` to `CheckUtils`, which sounds right to me.
Maybe we should also move the method `cannotBeNone()` the same way, WDYT ?

Reply:
I don't like to move methods to util classes that are only used in a single check. There are only a few cases where this is applicable. Eg a Utils class is only for a specific type of helpers like a dedicated InferredType utils class.